Makefile

```diff
@@ -64,10 +64,17 @@ build-mac-arm64:
 		-o bin/darwin/arm64/$(CLI_EXE) $(CLI_PKG)
 
 .PHONY: build-windows
-build-windows:
+build-windows: build-windows-amd64 build-windows-arm64
+
+.PHONY: build-windows-amd64
 	GOARCH=amd64 CGO_ENABLED=0 GOOS=windows go build -v --ldflags="$(LDFLAGS)" \
 		-o bin/windows/amd64/$(CLI_EXE).exe $(CLI_PKG)
 
+.PHONY: build-windows-arm64
+build-windows:
+	GOARCH=arm64 CGO_ENABLED=0 GOOS=windows go build -v --ldflags="$(LDFLAGS)" \
+		-o bin/windows/arm64/$(CLI_EXE).exe $(CLI_PKG)
```

Comment:
```suggestion
build-windows: build-windows-amd64 build-windows-arm64

.PHONY: build-windows-amd64
build-windows-amd64:
	GOARCH=amd64 CGO_ENABLED=0 GOOS=windows go build -v --ldflags="$(LDFLAGS)" \
		-o bin/windows/amd64/$(CLI_EXE).exe $(CLI_PKG)

.PHONY: build-windows-arm64
build-windows-arm64:
	GOARCH=arm64 CGO_ENABLED=0 GOOS=windows go build -v --ldflags="$(LDFLAGS)" \
		-o bin/windows/arm64/$(CLI_EXE).exe $(CLI_PKG)
```

Reply:
Added the suggestion. Please have a look